src/libcephfs.cc

```diff
@@ -184,6 +187,14 @@ struct ceph_mount_info
     return mounted;
   }
 
+
+  mode_t umask(mode_t mask)
+  {
+    DEFAULT_UMASK = mask;
```

Comment:
so this will be `this->mask = mask;`.

Reply:
You will need to add a callback function to `struct ceph_mount_info`. We can cover that today in our meeting.